Sources/Mailgun/Client.swift

```diff
@@ -6,9 +6,13 @@ import Models
 import Optics
 import PointFreePrelude
 import Prelude
+import Tagged
 import UrlFormEncoding
 
 public struct Client {
+  public typealias ApiKey = MailgunForwardPayload.ApiKey
```

Comment:
Because `Mailgun` depends on `Model`s, the tag needs to be there.

Reply:
do you think it would make sense to move `verify(with:)` from `MailgunForwardPayload.swift` to this file? Then `MailgunForwardPayload` wouldn't even need an `ApiKey` I don't think...